addressing/README.md

```diff
@@ -0,0 +1,337 @@
+# Addressing in libp2p
+> How network addresses are encoded and used in libp2p
+
+| Lifecycle Stage | Maturity      | Status | Latest Revision |
+|-----------------|---------------|--------|-----------------|
+| 1A              | Working Draft | Active | r0, 2019-05-27  |
```

Comment:
```suggestion
| 1A              | Recommendation | Active | r0, 2019-05-27  |
```

> ### Level 3: Recommendation
>
> There are at least TWO implementations conforming to the specification, with
demonstrated cross-interoperability.

This specification fulfills the requirements of _Recommendation (Level 3)_.